src/test/java/edu/hawaii/its/api/groupings/GroupingAllTest.java

```diff
@@ -0,0 +1,49 @@
+package edu.hawaii.its.api.groupings;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import edu.hawaii.its.api.type.GroupingPath;
+
+public class GroupingAllTest {
+
+    private GroupingPaths groupingAll;
+    private GroupingPaths groupingAll2;
+
+    @BeforeEach
+    public void setup() {
+        List<GroupingPath> groupingPaths = new ArrayList<>();
+        groupingPaths.add(new GroupingPath("path:to:grouping"));
+        groupingAll = new GroupingPaths();
+        groupingAll2 = new GroupingPaths(groupingPaths);
+    }
```

Comment:
Be sure to update the variables `groupingAll` and `groupingAll2` to something like `groupingPaths` and `groupingPaths2`. Then in` groupingPaths2`, you can directly pass in `Arrays.asList("path:to:grouping")` instead of creating a new variable. This is to avoid the confusion of the type of `groupingPaths` either being `GroupingPaths` or` List<GroupingPath>`. Thanks!